_topics/en/musig.md

```diff
@@ -28,9 +28,19 @@ primary_sources:
     - title: MuSig-DN paper
       link: https://eprint.iacr.org/2020/1057.pdf
 
+    - title: Original MuSig2 implementation (experimental)
+      link: https://github.com/BlockstreamResearch/secp256k1-zkp
+
+    - title: MuSig2 implementation in Libsecp256k1
+      link: https://github.com/bitcoin-core/secp256k1/issues/1479
+
 ## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
 ## true" to bold entry
 optech_mentions:
+  - title: Libsecp256k1 adds MuSig2 BIP340-compatible multisig as specified in BIP327
+    url: /en/newsletters/2024/10/11/#libsecp256k1-1479
+    date: 2024-10-11
```

Comment:
I dont think the date is needed for the newsletter entries

Reply:
Done